.eslintcache

```diff
@@ -0,0 +1 @@
stjs/helpers/getComposition.js":{"size":1268,"mtime":1550653581606,"hashOfConfig":"1g3k428","results":{"filePath":"/Users/reuben.searle/Sites/NB-1808/the-wall-templater-dustjs/helpers/getComposition.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/reuben.searle/Sites/NB-1808/the-wall-templater-dustjs/helpers/getPartyCompFromRoomData.js":{"size":736,"mtime":1550653581606,"hashOfConfig":"1g3k428","results":{"filePath":"/Users/reuben.searle/Sites/NB-1808/the-wall-templater-dustjs/helpers/getPartyCompFromRoomData.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/reuben.searle/Sites/NB-1808/the-wall-templater-dustjs/helpers/getRoomDescription.js":{"size":916,"mtime":1550653581607,"hashOfConfig":"1g3k428","results":{"filePath":"/Users/reuben.searle/Sites/NB-1808/the-wall-templater-dustjs/helpers/getRoomDescription.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/reuben.searle/Sites/NB-1808/the-wall-templater-dustjs/helpers/hash.js":{"size":518,"mtime":1550653581607,"hashOfConfig":"1g3k428","results":{"filePath":"/Users/reuben.searle/Sites/NB-1808/the-wall-templater-dustjs/helpers/hash.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/reuben.searle/Sites/NB-1808/the-wall-templater-dustjs/helpers/isCarParking.js":{"size":711,"mtime":1550653581607,"hashOfConfig":"1g3k428","results":{"filePath":"/Users/reuben.searle/Sites/NB-1808/the-wall-templater-dustjs/helpers/isCarParking.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/reuben.searle/Sites/NB-1808/the-wall-templater-dustjs/helpers/loadTemplate.js":{"size":944,"mtime":1550653581608,"hashOfConfig":"1g3k428","results":{"filePath":"/Users/reuben.searle/Sites/NB-1808/the-wall-templater-dustjs/helpers/loadTemplate.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/reuben.searle/Sites/NB-1808/the-wall-templater-dustjs/helpers/loop.js":{"size":899,"mtime":1550653581608,"hashOfConfig":"1g3k428","results":{"filePath":"/Users/reuben.searle/Sites/NB-1808/the-wall-templater-dustjs/helpers/loop.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/reuben.searle/Sites/NB-1808/the-wall-templater-dustjs/helpers/sortAndFilterHotels.js":{"size":2987,"mtime":1550653581608,"hashOfConfig":"1g3k428","results":{"filePath":"/Users/reuben.searle/Sites/NB-1808/the-wall-templater-dustjs/helpers/sortAndFilterHotels.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/reuben.searle/Sites/NB-1808/the-wall-templater-dustjs/helpers/sortByMapping.js":{"size":2274,"mtime":1550653581609,"hashOfConfig":"1g3k428","results":{"filePath":"/Users/reuben.searle/Sites/NB-1808/the-wall-templater-dustjs/helpers/sortByMapping.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/reuben.searle/Sites/NB-1808/the-wall-templater-dustjs/helpers/sortByNumber.js":{"size":796,"mtime":1550653581609,"hashOfConfig":"1g3k428","results":{"filePath":"/Users/reuben.searle/Sites/NB-1808/the-wall-templater-dustjs/helpers/sortByNumber.js","messages":[],"errorCount":0,"warningCount":0}}}
```

Comment:
need to git ignore this file, I tend to use: https://www.gitignore.io

https://www.gitignore.io/api/node,macos,sublimetext,visualstudiocode